README.md

```diff
@@ -76,6 +76,32 @@ To access the server once bedrock is running:
 2. The certificate warning is normal for development mode. Accept it and
    continue.
 
+## Bedrock Events
+
+List of emitted
+[Bedrock Events](https://github.com/digitalbazaar/bedrock#bedrockevents):
+
+- **bedrock-server.readinessCheck**
+  - Emitted before listening starts on any ports.
```

Comment:
Thanks for the documentation.  If it's possible to work in the connection to https://github.com/digitalbazaar/bedrock-health/ it would be good.  Doesn't need to hold this PR up though.